Annotation on requests

arlemi opened the pull request:
First go at the annotation spec docs. Let me know if I missed anything obvious, or not obvious! 

@SueSmith I've put a border on all still images, hope it's fine if GIFs don't have it. 🤔

SueSmith commented on src/pages/docs/postman/collections/commenting-on-collections.md:
```diff
@@ -27,64 +29,97 @@ You can leave comments on collections from the [Postman Dashboard](https://app.g
-1. From the Postman app or [Dashboard](https://app.getpostman.com), open the collection with the request you want to comment on.
-2. At the top right, click **Comments**.
-   * You'll also see any other comments on the request.
+### From the Postman Dashboard
```
if we could make this "Commenting from the Dashboard" plz

SueSmith commented on src/pages/docs/postman/collections/commenting-on-collections.md:
```diff
@@ -27,64 +29,97 @@ You can leave comments on collections from the [Postman Dashboard](https://app.g
+3. Click **Comments** next to the request name.
+4. Write your comment, then click **Add Comment**.
+
+### From the Postman app
```
and "Commenting in the app"

SueSmith commented on src/pages/docs/postman/collections/commenting-on-collections.md:
```diff
@@ -27,64 +29,97 @@ You can leave comments on collections from the [Postman Dashboard](https://app.g
+> You can only add comment to parameters on saved requests.
 
-![Comments in app and dashboard](https://assets.postman.com/postman-docs/Comment+on+app+and+dashboard.gif)
+1. Open the request you'd like to comment.
```
"comment" > "comment on"

SueSmith commented on src/pages/docs/postman/collections/commenting-on-collections.md:
```diff
@@ -27,64 +29,97 @@ You can leave comments on collections from the [Postman Dashboard](https://app.g
 ## Reading comments in the app and on the Dashboard
 
-You can see comments made by teammates on requests and collections in the Postman app or the [Dashboard](https://app.getpostman.com).
+You can see comments made by teammates on requests and request parameters in the Postman app or the [Dashboard](https://app.getpostman.com). Comments on collections can only be accessed from the Dashboard.
```
if we could just reword the second sentence to make it active "You can only access comments on collections from the Dashboard."

SueSmith commented on src/pages/docs/postman/collections/commenting-on-collections.md:
```diff
@@ -27,64 +29,97 @@ You can leave comments on collections from the [Postman Dashboard](https://app.g
 
 ### Reading comments in the Postman app
 
+You can read both comments made on the request and on request parameters from the app.
```
I'd lose "both" here

SueSmith commented on src/pages/docs/postman/collections/commenting-on-collections.md:
```diff
@@ -27,64 +29,97 @@ You can leave comments on collections from the [Postman Dashboard](https://app.g
+
 1. In the Postman app, open the request that has the comments you want to review.
 2. At the top, click **Comments**.
+3. Select if you'd like to see comments made on the request (**Comments**) or on the request parameters (**Inline Comments**)
```
this is an awkward one to word, think I'd just go "Select comments made on the request...." and we're missing a period at the end of the sentence

SueSmith commented on src/pages/docs/postman/collections/commenting-on-collections.md:
```diff
@@ -27,64 +29,97 @@ You can leave comments on collections from the [Postman Dashboard](https://app.g
 ### Reading comments on the Dashboard
 
-You can read the comments on a collection through the [Postman Dashboard](https://app.getpostman.com).
+You can read all type of comments through the [Postman Dashboard](https://app.getpostman.com).
```
"all comments" should be fine here

SueSmith commented on src/pages/docs/postman/collections/commenting-on-collections.md:
```diff
@@ -27,64 +29,97 @@ You can leave comments on collections from the [Postman Dashboard](https://app.g
 2. Find the comment and click **Comment** > **Edit**.
 
-![Edit comment](https://assets.postman.com/postman-docs/Edit+comments+in+Postman.jpg)
+![Edit comment](https://assets.postman.com/postman-docs/collab09-edit-comment.jpg)
```
image is a tiny bit pixelated probably because I'm on this daft mac - fyi although it isn't ideal we can use <img> tags and specify width in cases where the image renders better resized down..

arlemi replied:
I've forced it with html, let me know if that fixes it.

SueSmith commented on src/pages/docs/postman/collections/commenting-on-collections.md:
```diff
@@ -27,64 +29,97 @@ You can leave comments on collections from the [Postman Dashboard](https://app.g
+
+You can resolve comments or threads of comments made on request parameters.
+
+> Resolving threads is only possible through the app
```
wee full stop missing here

SueSmith commented on src/pages/docs/postman/collections/commenting-on-collections.md:
```diff
@@ -27,64 +29,97 @@ You can leave comments on collections from the [Postman Dashboard](https://app.g
+3. Click **Inline Comments**.
+4. Click **Resolve** next to the thread you'd like to resolve.
+
+GIF resolving comments
```
some extra text here 🙃

arlemi replied:
oopsie

sankalp0o commented:
@arlemi this looks good! 

Two observations
1. We are saying "Request parameters", but you can actually create inline comments on params, headers, request body of type form-data and x-www-form-urlencoded.
2. Can we review what we call "thread" and instead say "inline comments"? I think we don't use the term "thread" in the product, for e.g. "All the inline comments for this request have been resolved..."

arlemi commented:
@SueSmith Adressed your comments, please double check the image and let me know if I should try to take it at a higher res. 

@sankalp0o I've replaced all mentions of thread by inline comment / inline comments. Also updated what we mean by request parameters in: 
> You can leave comments on request parameters (query parameters, path parameters, headers, request body of type form-data, and x-www-form-urlencoded) from the Postman app.

Let me know if that works.

SueSmith commented on src/pages/docs/postman/collections/commenting-on-collections.md:
```diff
@@ -27,64 +29,96 @@ You can leave comments on collections from the [Postman Dashboard](https://app.g
 ## Commenting on a request
 
-You can comment on requests from the [Postman Dashboard](https://app.getpostman.com) or in the Postman app. When you comment on a request, it's reflected in both the Dashboard and the app.
+You can comment on requests [from the Postman Dashboard](#from-the-postman-dashboard) or [in the Postman app](#from-the-postman-app). When you comment on a request, it's reflected in both the Dashboard and the app.
```
should have mentioned this last time but these links will need updated since the headings changed 😆

SueSmith commented on src/pages/docs/postman/collections/commenting-on-collections.md:
```diff
@@ -27,64 +29,96 @@ You can leave comments on collections from the [Postman Dashboard](https://app.g
 2. Find the comment and click **Comment** > **Edit**.
 
-![Edit comment](https://assets.postman.com/postman-docs/Edit+comments+in+Postman.jpg)
+<img src="https://assets.postman.com/postman-docs/collab09-edit-comment.jpg" alt="Edit comment" width="476px"/>
```
aye it's still rendering pixelated to me

arlemi commented:
@SueSmith I've updated the images, if you're not seeing it straight away it may be S3 taking some time to update. 

@sankalp0o I've added information about the feature being in beta.

arlemi commented:
@SueSmith FYI - this won't be released in 7.26, I'll hold off on merging until we release it.

arlemi commented:
We're releasing an updated version of this, since develop changed quite a lot since I'll create a new PR.
